salt/sciencebeam-texture/init.sls

```diff
@@ -6,6 +6,8 @@ sciencebeam-texture-docker-compose-folder:
         - recurse:
             - user
             - group
+        - require:
```

Comment:
this state doesn't appear to have anything to do with `/ext` (unless there is a symlink hiding in there?)

it might be more comfortable attached to `sciencebeam-texture-docker-containers` if the app configuration at that point has some reference `/ext`

Reply:
I've just done some config spelunking and we have `sciencebeam-texture-docker-compose-.env` rendering a template that is calling custom python code that starts a subprocess to execute a bash script that executes docker commands. crazy stuff. 

anyway, I'll move this requisite to `sciencebeam-texture-docker-compose-.env`

Reply:
![common-sense-just-because-you-can-doesn-t-mean-you-should](https://user-images.githubusercontent.com/2142748/62178282-a9120c80-b387-11e9-9831-6a70493437b9.jpg)